Dockerfile

```diff
@@ -1,22 +1,38 @@
-FROM node:7.8.0
+FROM node:7.8.0-alpine as nodebuild
 
-# Install runtime dependencies and remove state generated by apt-get before
-# creating intermediate image.
-RUN apt-get update && apt-get install -y mediainfo && rm -rf /var/lib/apt/lists/*
+# Generate node_modules.
+WORKDIR /tmp
+COPY package.json /tmp/package.json
+COPY package-lock.json /tmp/package-lock.json
+RUN apk --no-cache add python build-base
+RUN npm install
+
+# bcrypt needs to be rebuilt for musl
+RUN npm rebuild bcrypt --build-from-source
+
+################################################################################
+FROM node:7.8.0-alpine
+
+# Install runtime dependencies.
+RUN apk --no-cache \
+       --repository http://dl-cdn.alpinelinux.org/alpine/v3.7/community \
```

Comment:
Hard-coding the alpine repository version maybe has the potential to cause issues when the image updates the Alpine version. Is specifying the repository here even necessary?

Reply:
No hard coding URL or this will be unmaintainable .

Reply:
@wopian There is no hard-coded version in what you suggest but why the `--repository` is even needed ?

Reply:
The community repo is available by default in non-edge builds of alpine making line 18 redundant